docs/classic-ui/forms.md

```diff
@@ -14,4 +14,90 @@ myst:
 ```{todo}
 Describe how create forms with Plone's default form framework z3c.form.
 Fields, Widgets, Vocabularies aso are descripted in detail in there own chapter and will be referenced frome examples here.
-```
\ No newline at end of file
+```
+
+Plone uses the [z3c.form](http://pythonhosted.org/z3c.form) library to build its forms, via the [plone.z3cform](http://pypi.python.org/pypi/plone.z3cform) integration package.
+
+For an easy way to structure a form and define its widgets and fields, Plone relies on [plone.autoform](http://pypi.python.org/pypi/plone.autoform), in particular its AutoExtensibleForm base class.
+Which is responsible for processing form hints and setting up [z3c.form](http://pythonhosted.org/z3c.form) widgets and groups (fieldsets).
```

Comment:
```suggestion
To simplify the process of organizing a form and specifying its widgets and fields, Plone utilizes [plone.autoform](http://pypi.python.org/pypi/plone.autoform), in particular its `AutoExtensibleForm` base class.
It is responsible for handling form hints and configuring  z3c.form widgets and groups (fieldsets).
```
IMO easier to read this way, also z3c.form was already linked in the introductory sentence.

Do we have form-hints aka directives documented already somewhere? If yes, we need a reference, if not we need a todo/an issue.